lib/ansible/plugins/action/__init__.py

```diff
@@ -747,7 +749,7 @@ def _low_level_execute_command(self, cmd, sudoable=True, in_data=None, executabl
         out = self._strip_success_message(out)
 
         display.debug("_low_level_execute_command() done: rc=%d, stdout=%s, stderr=%s" % (rc, stdout, stderr))
-        return dict(rc=rc, stdout=out, stdout_lines=out.splitlines(), stderr=err)
+        return dict(rc=rc, stdout=out, stdout_lines=out.splitlines(), stderr=err, stderr_lines=err.splitlines())
```

Comment:
Sorry, that was a bit "dense". :) The test that's failing is because the stderr object that comes back is not guaranteed to be something that has splitlines() defined (in this test, for whatever reason, it comes back as a bool). It isn't for stdout either, but all the connection plugins have been normalized to fix that (otherwise this would've blown up before on stdout). 

In Python it's actually cheaper to just call splitlines() and catch the exception if it's not there- I'd suggest setting the stderr_lines value to ```[ str(stderr) ]``` in that case.